manifest_staging/charts/gatekeeper/values.yaml

```diff
@@ -145,6 +145,7 @@ controllerManager:
   priorityClassName: system-cluster-critical
   disableCertRotation: false
   tlsMinVersion: 1.3
+  certName: ""
```

Comment:
This will need to be set here in order to prevent this value from being clobbered:

https://github.com/open-policy-agent/gatekeeper/blob/master/cmd/build/helmify/static/values.yaml

see https://open-policy-agent.github.io/gatekeeper/website/docs/help/#contributing-to-helm-chart for more info